Request/RequestBodyParamConverter20.php

```diff
@@ -16,7 +16,7 @@
 
 /**
  * This code is needed for SensioFrameworkExtraBundle 2.x compatibility
- * https://github.com/FriendsOfSymfony/FOSRestBundle/issues/622
+ * https://github.com/FriendsOfSymfony/FOSRestBundle/issues/622.
```

Comment:
Oups, I didn't see this (Php-cs-fixer changes everything).
I think it's better to leave this to another PR, to discuss if we should support sensio 2.x still.
Unless you are sure that we can remove that ?